roles/openshift_node/tasks/dnsmasq_install.yml

```diff
@@ -12,22 +12,23 @@
 
 - when: not openshift_is_atomic | bool
   block:
-  - name: Ensure dbus is updated before installing dnsmasq
-    package:
-      name: dbus
-      state: latest
-    register: dbus_update
-  - name: Restart dbus if it was updated
-    systemd:
-      name: dbus
-      state: restarted
-    when: dbus_update | changed
   - name: Install dnsmasq
     package:
       name: dnsmasq
       state: installed
     register: result
     until: result is succeeded
+  # This works around https://bugzilla.redhat.com/show_bug.cgi?id=1550582
+  - name: Restart dbus and systemd-logind if dnsmasq was installed
+    systemd:
+      name: "{{ item }}"
+      state: restarted
+    when: result is changed
+    with_items:
+    - dbus
+    - systemd-logind
+    register: dbussvcs_restart
```

Comment:
Do we need to register this?

Reply:
I was going to make wait_for_connection conditional on this, but then I decided it was harmless. I do wonder if we should add in a second or two delay?

Reply:
I'm not sure.  Does restarting systemd-logind bounce a connection?  I would assume the ssh session would just be resumed.  I'm unsure how this would behave.